### ISSUE TITLE
About FLOPs

### ISSUE THREAD

**Issue by lld533** [d-li14/HBONet#4]
Hi,

I'm calculating the FLOPs of the default settings of HBOnet where width_mult = 1.0 and input size is 3x224x224 with thop (https://github.com/Lyken17/pytorch-OpCounter). Below please find my script with Python3.5.

import hbonet
from thop import profile, clever_format
import torch

model = hbonet.hbonet()
flops, params = profile(model, inputs=(torch.randn(1,3,224,224), ))
flops, params = clever_format([flops, params], "%.3f")
print(flops, params) #984.760M 4.562M

It turns out to me that FLOPs and params are 984.760M & 4.562M, respectively. As you can see, the number of flops is far more than 300M claimed in your paper. 

May I know if you counted nn.Upsample and nn.BatchNorm2d in your paper? To make it clear, could you please release the code you used to count FLOPs in your paper? Many thanks in advance.

**Comment by d-li14** [comment]
In my case, the toolbox you mentioned outputs wrong results frequently. I prefer https://github.com/nmhkahn/torchsummaryX. You could also check it by yourself to see which one is more precise. BTW, upsample and BN layers are usually not included according to the definition of FLOPs.
